corehq/apps/app_manager/fields.py

```diff
@@ -160,10 +160,17 @@ class ApplicationDataRMIHelper(object):
     APP_TYPE_NONE = 'no_app'
     APP_TYPE_UNKNOWN = 'unknown'
 
-    def __init__(self, domain, user, as_dict=True, form_placeholders=None, case_placeholders=None):
+    def __init__(self, domain, user, as_dict=True, form_labels=None,
```

Comment:
any reason to add `form_labels` in the middle here instead of at the end? usually it's safer to add a new kwarg to the end unless you know all everything that calls this doesn't use positional args

Reply:
makes sense, updated